src/org/thoughtcrime/securesms/ConversationFragment.java

```diff
@@ -167,22 +168,23 @@ private void setNoMessageText() {
                 noMessageTextView.setText(R.string.chat_no_messages);
             }
         }
-        else if(getListAdapter().isGroupChat()){
+        else if(dcChat.isGroup()){
             if(dcContext.getChat((int) chatId).isUnpromoted()) {
```

Comment:
I'm not sure, but is `dcContext.getChat((int) chatId)` the same here as `dcChat` from line 162?

Reply:
i think so. the ListAdapter contains a DcChat object and some getters that are using that.
i've added a ListAdapter.getChat() function to avoid duplicating the whole DcChat api - and also we can avoid a database roundtrip this way.

so getListAdapter().isGroupChat() calls isGroup() on the internal chat object - this is the same as calling getChat().isGroup(),